src/mdx-pages/guides/options.mdx

```diff
@@ -615,6 +615,20 @@ Override the mute key definition. If this is set, the function receives the `key
 
 Override the play/pause key definition. If this is set, the function receives the `keydown` event; if the function returns `true`, then the play/pause toggle action is performed.
 
+### `experimentalSvgIcons`
```

Comment:
I have a question about the position of this option. I noticed that the other options were arranged in alphabetical order. Should there be some consistency? Or was this done on purpose because it's an experimental option?